### PR TITLE
Expand card status enum

### PR DESCRIPTION
**Pull request by UgurGumushan**
Screenshot 2023-07-18 at 17 25 39

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kontist/mock-solaris",
-  "version": "1.0.87",
+  "version": "1.0.88-b",
```
Comment (p-janik) [inline package.json]: Is it some kind of experimental convention?

I think in case of enum extension we can simply go with standard patch version bump?

Comment (UgurGumushan) [reply package.json]: I needed this to experiment/run tests on the backend (with ease)
I've reverted it and we can release it after merge

**Review by p-janik — APPROVED** [review]

**Comment by p-janik** [comment]
I'm wondering if its used in any way here? Just extending this enum also extends the states which we can select in cards tab?

**Comment by UgurGumushan** [comment]
> I'm wondering if its used in any way here? Just extending this enum also extends the states which we can select in cards tab?

The type is imported from the backend and used. 
I needed to extend it on this so that the types match with the extended card status type on the backend. 
Otherwise I had compilation errors.
I didn't extend the select box as we don't have a practical use for it yet. 
(Apps don't respond to changes to those card statuses)

**Comment by UgurGumushan** [comment]
These types are behind a feature flag on the solaris side and are not used in our api calls yet. Closing.

**Pull request closed without merging**